includes/definitions/routeros.yaml

```diff
@@ -1,15 +0,0 @@
-os: routeros
```

Comment:
Why have you deleted this file?

Reply:
That was when I had modified it when creating the second second OS, as that wasn't needed I deleted that file so it would not be included in the PR

Reply:
Without that file, routeros will not work at all. You'll need to restore it.